src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -58,7 +58,16 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <None Include="Data\CodePageNameMappings.csv" />
```

Comment:
Personally, I dislike "invisible" source files.  It was more necessary for automatic building, though.